### PR TITLE
[velero] Correct syntax of iam role pod annotation in values.yaml

### PR DESCRIPTION
**Pull request by mukulgit123**
Signed-off-by: mukul.garg <mukul.garg@ymail.com>

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[velero]`)

### CODE REVIEW

File: charts/velero/values.yaml [hunk 1]
```diff
@@ -19,8 +19,9 @@ image:
 #
 # If using kube2iam or kiam, use the following annotation with your AWS_ACCOUNT_ID
 # and VELERO_ROLE_NAME filled in:
-#  iam.amazonaws.com/role: arn:aws:iam::<AWS_ACCOUNT_ID>:role/<VELERO_ROLE_NAME>
+#  iam.amazonaws.com/role: "arn:aws:iam::<AWS_ACCOUNT_ID>:role/<VELERO_ROLE_NAME>"
 podAnnotations: {}
+  #  iam.amazonaws.com/role: "arn:aws:iam::<AWS_ACCOUNT_ID>:role/<VELERO_ROLE_NAME>"
```
Comment (jenting) [inline charts/velero/values.yaml:24]: I think we should provide one example in a single line is enough, please choose to keep line 22 or line 24.

Comment (mukulgit123) [reply charts/velero/values.yaml:23]: I have added the example on Line 24 and removed Line 22.

Comment (jenting) [reply charts/velero/values.yaml:23]: thanks ^^

**Comment by jenting** [comment]
@mukulgit123 
thanks for your contribution~
I have no experience in setup AWS IAM Role to Pod annotation, so I'm not sure the quoted is required or not.

I try to google some examples but all are without quoted:
- https://github.com/FairwindsOps/iam-waiter#usage
- https://stackoverflow.com/questions/58523869/spring-data-flow-iam-role-assignment-to-pods-using-pod-annotations

could you please check it or elaborate more clues on it? thanks.

**Comment by mukulgit123** [comment]
This was causing few issues when I tried without quotes and I was getting an error as below and it worked after I deployed with quotes on it. That's when I decided to raise a PR and help others. :)

_**install.go:189: [debug] WARNING: This chart or one of its subcharts contains CRDs. Rendering may fail or contain inaccuracies.
Error: YAML parse error on velero/templates/deployment.yaml: error converting YAML to JSON: yaml: line 26: mapping values are not allowed in this context
helm.go:94: [debug] error converting YAML to JSON: yaml: line 26: mapping values are not allowed in this context
YAML parse error on velero/templates/deployment.yaml
helm.sh/helm/v3/pkg/releaseutil.(*manifestFile).sort
	/private/tmp/helm-20200901-67641-iy9ufs/pkg/releaseutil/manifest_sorter.go:146
helm.sh/helm/v3/pkg/releaseutil.SortManifests
	/private/tmp/helm-20200901-67641-iy9ufs/pkg/releaseutil/manifest_sorter.go:106
helm.sh/helm/v3/pkg/action.(*Configuration).renderResources
	/private/tmp/helm-20200901-67641-iy9ufs/pkg/action/action.go:162
helm.sh/helm/v3/pkg/action.(*Install).Run
	/private/tmp/helm-20200901-67641-iy9ufs/pkg/action/install.go:239
main.runInstall
	/private/tmp/helm-20200901-67641-iy9ufs/cmd/helm/install.go:242
main.newInstallCmd.func2
	/private/tmp/helm-20200901-67641-iy9ufs/cmd/helm/install.go:120
github.com/spf13/cobra.(*Command).execute
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:842
github.com/spf13/cobra.(*Command).ExecuteC
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950
github.com/spf13/cobra.(*Command).Execute
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
main.main
	/private/tmp/helm-20200901-67641-iy9ufs/cmd/helm/helm.go:93
runtime.main
	/usr/local/Cellar/go/1.15/libexec/src/runtime/proc.go:204
runtime.goexit
	/usr/local/Cellar/go/1.15/libexec/src/runtime/asm_amd64.s:1374**_

**Review by jenting — APPROVED** [review]
LGTM, thanks for the explanation @mukulgit123

**Comment by jenting** [comment]
@cpanato @carlisia could you please take a look at this? thank you.

**Review by cpanato — DISMISSED** [review]
thanks for this fix!

**Comment by jenting** [comment]
Bump chart version.

**Review by jenting — APPROVED** [review]
LGTM

**Comment by jenting** [comment]
@cpanato, could you please help review it again? thank you 😃

**Review by cpanato — APPROVED** [review]

**Pull request merged**